test/dlc_tests/ec2/mxnet/training/test_mxnet_training.py

```diff
@@ -16,6 +18,9 @@
 
 MX_EC2_GPU_INSTANCE_TYPE = get_ec2_instance_type(default="g3.8xlarge", processor="gpu")
```

Comment:
should we use a p type instance?

Reply:
g3.8 instances are cheaper than most multi-gpu p3 instances, which is why we moved to `g3.8xlarge`. Do you recommend we use a p-type instance here, for any reason?

Reply:
The test corresponding this variable was using `p2.xlarge`. If `g3.8` is cheaper and achieves the same goal then we can keep this configuration.

Reply:
The tests that used `g3.8xlarge` by default before this will continue to use the same instance type.

The test that used `p2.xlarge` is changed to use `p3.2xlarge` by default because that is the smallest p3 instance type.